### ISSUE TITLE
Add aria label and status to EditInPlace

### ISSUE THREAD

**Issue by elycheea** [carbon-design-system/ibm-products#4467]
Reported during the [EditInPlace review](https://github.ibm.com/CDAI-design/pal/issues/1487) of the accessibility audit.

## What will this achieve?

Improves accessibility of EditInPlace by resolving two violations.

- because this component doesn’t use a visible label the input needs to use `aria-label`
- the input also needs to make use of `aria-invalid` when in the invalid state

## How will success be measured?

Remove ARIA violations on EditInPlace.

## Additional information

- [Edit-in-place accessibility review](https://github.ibm.com/CDAI-design/pal/issues/1487)

```[tasklist]
### Acceptance criteria
- [ ] Require an `aria-label` (or use `labelText`) as default
- [ ] Add `aria-invalid` when input is `invalid`
```

**Comment by elycheea** [comment]
Believe we can either add `aria-hidden` to the current label or move `labelText` into `aria-label`.
